go.mod

```diff
@@ -526,7 +526,7 @@ replace (
 	k8s.io/klog => k8s.io/klog v1.0.0
 	k8s.io/kube-aggregator => ./staging/src/k8s.io/kube-aggregator
 	k8s.io/kube-controller-manager => ./staging/src/k8s.io/kube-controller-manager
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf
+	k8s.io/kube-openapi => github.com/liggitt/kube-openapi v0.0.0-20200409184531-690c19a814e8
```

Comment:
🙈

Reply:
I'm making sure CI is green here before unholding https://github.com/kubernetes/kube-openapi/pull/195